controllers/TasksController.php

```diff
@@ -74,7 +115,46 @@ public function actionView(int $id)
                 'responds' => $responds,
                 'category' => $category,
                 'deadline' => $deadline,
+                'files' => $files,
             ]
         );
     }
+
+    /**
+     * Страница с формой создания задания
+     * 
+     * @return string - код страницы с формой создания задания
+     */
+    public function actionCreate()
+    {
+        $user = Yii::$app->user->getIdentity();
+
+        if (User::ROLE_EXECUTOR === $user->user_role) {
```

Comment:
Тут лучше проверять на неравенство с конкретной ролью. Если в будущем в проекте появятся новые роли, то создание задания будет запрещено только роли "Исполнитель"

```suggestion
        if ($user->user_role !== User::ROLE_CUSTOMER) {
```